src/main/java/org/sagebionetworks/bridge/spring/controllers/StudyController.java

```diff
@@ -174,19 +175,28 @@ public Study finishStudyLogo(@PathVariable String id, @PathVariable("createdOn")
     }
 
     @GetMapping(path = "/v1/apps/{appId}/studies/{studyId}", produces = { APPLICATION_JSON_UTF8_VALUE })
```

Comment:
Separate this out into a /v2/ URL for the Worker API.

Add a comment about why v1 exists. This is used to show app participants some basic study info before they create their accounts.